sunpy/net/helioviewer.py

```diff
@@ -132,11 +132,17 @@ def get_closest_image(self, date, sourceid):
 
         return response
 
-    def download_jp2(self, date, sourceid, directory=None, overwrite=False, **kwargs):
+    def display_source_id(self, observatory, instrument, detector, measurement):
+        """
+        Returns a dictionary containing the source ID of all the 
+        images in the data source
+        """
+        return self._source_dict[(observatory, instrument, detector, measurement)]
+
+    def download_jp2(self, date, directory=None, overwrite=False, **kwargs):
```

Comment:
You will need to add back the old API here as well.

Reply:
I'm sorry I didn't get that. Where do you want me to add the older API